test_scripts/API/AutoCompleteList/004_ATF_SetGlobalProperties_P_AutoCompleteList_ValueUpperBound.lua

```diff
@@ -0,0 +1,118 @@
+----------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = true, resultCode = "SUCCESS"> to mobile app
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList>  array is upper bound.
+-- 3. SDL respond <success = true, resultCode = "SUCCESS"> to mobile app
+----------------------------------------------------------------------------------------
+---[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+
+--[[ General Precondition before ATF start ]]
+config.defaultProtocolVersion = 2
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_AutoCompleteList_Array_Is_UpperBound()
+--mobile side: sending SetGlobalProperties request
+local cid = self.mobileSession:SendRPC("SetGlobalProperties",
+    {
+      keyboardProperties =
+      {
+        keyboardLayout = "QWERTY",
+        keypressMode = "SINGLE_KEYPRESS",
+        limitedCharacterList =
+        {
+          "a"
+        },
+        language = "EN-US",
+        autoCompleteText = "Text_1, Text_2",
+        autoCompleteList ={"TestList_100","TestList_100","TestList_100","TestList_100","TestList_100",
```

Comment:
@Aleksandriys, plase do not use such big constructions in requests in test like autoCompleteList there.
You may create autoCompleteList array with needed size in --[[ Local variables ]] using, for example:

autoCompleteList ={}
for i=1, 'upperBoundValue' do
  autoCompleteList[i]=<array_element>
end

and after that use it in requests.